azure-pipelines.yml

```diff
@@ -156,6 +156,26 @@ steps:
       BUILD_ID: $(Build.BuildId)
     workingDirectory: '$(System.DefaultWorkingDirectory)'
 
+  # TODO: There is no way to run steps in parallel in Azure pipelines but ideally this step would run in parallel
+  # TODO: with the above testing step to reduce overall runtime
+  - script: |
```

Comment:
Maybe we should just make a separate `envtest` context, and set it up with different config options, and then run the two of those in parallel?

Reply:
We talked through some options for this but won't do them in this PR.